src/business_logic/execution/execution_entry_point.rs

```diff
@@ -302,4 +242,87 @@ impl ExecutionEntryPoint {
 
         get_deployed_address_class_hash_at_address(state, &code_address.unwrap())
     }
+
+    fn _execute_version0_class<'a, T>(
```

Comment:
Is this prefixed by an underscore for any specific reason or was it just the name in cairo-lang?

Reply:
It was the name in cairo-lang